client/src/pages/resources/components/create-resource.tsx

```diff
@@ -11,6 +12,13 @@ import {
 
 export default function CreateResource() {
   const { resourceType } = useParams() as { resourceType: string }
+  const [sslRequired, setSslRequired] = useState(false)
```

Comment:
Instead of managing our own states, we can leverage the `Form` component's `getFieldValue` method.

Reply:
Already tried. doesn't work with checkbox

Reply:
Issue with form update based on checkbox value